pulsar_spectra/catalogue_papers/Deller_2009.yaml

```diff
@@ -1,21 +1,21 @@
 {
  "J0108-1431": {
   "Frequency MHz": [
-   1650.0
```

Comment:
I used different frequency values based on the text rather than the table "All observations were made at 1650 MHz, with the exception of PSR J0437−4715 which was observed separately at 8400 MHz". Please revert

Reply:
OK, I checked it, you are right.

Reply:
Since you agree, please change back to 1650 MHz

Reply:
The paper mentions flux density in a frequency of 1600 MHz, not 1650 MHz. I am confused about whether the central frequency is 1600 or 1650.

Reply:
Many papers round to common frequencies in the tables so I'm going to revert the frequency to 1650 MHz